Makefile

```diff
@@ -428,25 +428,75 @@ validate-codeserver-image: bin/kubectl
 		fi; \
 	done ; \
 
+# R Script (test.R)
+define R_SCRIPT
+	# Set a seed for reproducibility
+	set.seed(123)
+
+	# Generate a random dataset
+	x <- rnorm(100)
+	data <- data.frame(
+	x = x,
+	y = 2 * x + rnorm(100)
+	)
+	if ("x" %in% colnames(data) && is.numeric(data$x) &&
+		"y" %in% colnames(data) && is.numeric(data$y)) {
+	# Compute basic statistics
+	mean_x <- mean(data$x, na.rm = TRUE)
+	mean_y <- mean(data$y, na.rm = TRUE)
+	correlation <- cor(data$x, data$y, use = "complete.obs")
+	# Print the statistics
+	cat("Mean of x:", mean_x, "\n")
+	cat("Mean of y:", mean_y, "\n")
+	cat("Correlation between x and y:", correlation, "\n")
+
+	} else {
+	cat("Error: Columns 'x' and 'y' must be numeric and exist in the dataset.\n")
+	}
+endef
+
+export R_SCRIPT
+
 .PHONY: validate-rstudio-image
 validate-rstudio-image: bin/kubectl
 	$(eval NOTEBOOK_NAME := $(subst .,-,$(subst cuda-,,$*)))
-	$(info # Running tests for $(NOTEBOOK_NAME) code-server image...)
+	$(info # Running tests for $(NOTEBOOK_NAME) RStudio image...)
 	$(KUBECTL_BIN) wait --for=condition=ready pod rstudio-pod --timeout=300s
 	@required_commands=$(REQUIRED_R_STUDIO_IMAGE_COMMANDS) ; \
 	if [[ $$image == "" ]] ; then \
 		echo "Usage: make validate-rstudio-image image=<container-image-name>" ; \
 		exit 1 ; \
 	fi ; \
+	echo "=> Checking container image $$image for package intallation..." ; \
+	$(KUBECTL_BIN) exec -it rstudio-pod -- mkdir -p /opt/app-root/src/R/temp-library > /dev/null 2>&1 ; \
+	$(KUBECTL_BIN) exec rstudio-pod -- R -e "install.packages('tinytex', lib='/opt/app-root/src/R/temp-library')" > /dev/null 2>&1 ; \
+	if [ $$? -eq 0 ]; then \
+        echo "Tinytex installation successful!"; \
+    else \
```

Comment:
The indentation is strange. Shouldn't the else be aligned under the if to which it belongs?

Reply:
It is shown like this only on git preview, if you clone the PR it is fine, could you evaluate from you side as well?

Reply:
https://patch-diff.githubusercontent.com/raw/opendatahub-io/notebooks/pull/392.diff looks wrong too,

![image](https://github.com/opendatahub-io/notebooks/assets/442720/9e3a886d-66b2-44d4-891f-438abcf869cc)

ok, I'll go clone this and hopefully it will align itself better then

Reply:
Let us know, this how it looks on my vs code
![image](https://github.com/opendatahub-io/notebooks/assets/42587738/3aeb1134-6b8b-423e-a263-4fd8bc9f57d8)

Reply:
I see the same, but my background color is much nicer than yours ;ddd

![image](https://github.com/opendatahub-io/notebooks/assets/442720/7b54bda8-892e-40a1-a55d-9da5c69e8ddb)

What #!^% is going on with the GitHub previews?!

Reply:
Got it, mixing tabs and spaces, and configured tab size!

![image](https://github.com/opendatahub-io/notebooks/assets/442720/b7ec17ea-2673-47c0-a833-687589b73d5b)

Is it possible to be more consistent regarding tabs/spaces?

Reply:
The mystery solved!

Reply:
nice, thanks!